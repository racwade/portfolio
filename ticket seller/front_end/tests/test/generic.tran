00 username            SS 000009999