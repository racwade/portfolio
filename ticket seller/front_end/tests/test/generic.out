welcome to the ticket seller system
enter username:
bye